qutebrowser/browser/qutescheme.py

```diff
@@ -158,6 +159,69 @@ def qute_bookmarks(_url):
     return 'text/html', html
 
 
+@add_handler('history')
+def qute_history(url):
+    """Handler for qute:history. Display history."""
+
+    # Get current date from query parameter, if not given choose today.
+    curr_date = datetime.date.today()
+    try:
+        for query in url.query().split("&"):
+            kv = query.split("=", 2)
+            if kv[0] == "date":
+                curr_date = datetime.datetime.strptime(kv[1], "%Y-%m-%d")
+                break
+    except Exception:
+        log.misc.debug("Invalid query string passed to qute://history")
+
+    ONE_DAY = datetime.timedelta(days=1)
+    next_date = curr_date + ONE_DAY
+    prev_date = curr_date - ONE_DAY
+
+    def history_iter():
+        for item in objreg.get('web-history').history_dict.values():
+            # Convert timestamp
+            item_date = None
```

Comment:
If this couldn't be set we continue anyways, so no point in setting this to `None` here.